std_msgs/UInt16MultiArray dxlIDs
uint16 startAddress
std_msgs/UInt16MultiArray isWord
---
std_msgs/UInt16MultiArray values
bool txSuccess
